common/src/main/java/org/conscrypt/AddressUtils.java

```diff
@@ -56,7 +56,7 @@ public static boolean isValidSniHostname(String sniHostname) {
     /**
      * Returns true if the supplied hostname is an literal IP address.
      */
-    public static boolean isLiteralIpAddress(String hostname) {
+    static boolean isLiteralIpAddress(String hostname) {
```

Comment:
ah ok.  We should be more strict about our public API, however.  Perhaps we should introduce an `@InternalApi` annotation? WDYT?